scripts/github-release.sh

```diff
@@ -64,20 +64,20 @@ sed_inplace "--version=$OLD_VERSION" "--version=$RELEASE_VERSION" "$WORKSPACE_DI
 
 # Build the deploy.yaml
 make build-deploy
-cp "deploy.yaml" "examples/deploy-$RELEASE_VERSION.yaml"
+cp "deploy.yaml" "files/controller-installation/deploy-$RELEASE_VERSION.yaml"
```

Comment:
Wondering if we can figure out a more descriptive name than "files" here. Should we just move these two directories to top-level? So we have `/examples` and `/controller-installation`?

Reply:
we could but it’s a lot of work and if I miss a dependency some command will not work. is this a must or a nice to have?